src/Apprenticeships/SFA.DAS.Apprenticeships/Application/Notifications/NotificationCommandHandlerBase.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MediatR;
+
+namespace SFA.DAS.Apprenticeships.Application.Notifications
```

Comment:
Do these 3 classes (this, NotificationResponse, Recipient) want to be inside the Application/Notifications folder?